maintainers/maintainer-list.nix

```diff
@@ -16346,12 +16346,6 @@
       fingerprint = "79A1 CC17 67C7 32B6 A8A2  BF4F 71E0 8761 642D ACD2";
     }];
   };
-  samuelrivas = {
-    email = "samuelrivas@gmail.com";
-    github = "samuelrivas";
-    githubId = 107703;
-    name = "Samuel Rivas";
-  };
```

Comment:
I am not sure about this change.  I know some inactive maintainers are removed, however, I think this person is still [active](https://github.com/NixOS/nixpkgs/pulls?q=is%3Apr+author%3Asamuelrivas+).

Reply:
The most recent commit was two weeks ago. I have written this patch before that.
And before that the latest change of code was two years ago.

The best I can do now is not to remove him from maintainers list.

Reply:
Have you asked this person for his opinion? If not, I'd suggest doing it in a different pr and trying to contact him.

Reply:
Let's do this here.